databases/mysql57/Portfile

```diff
@@ -161,15 +160,6 @@ if {$subport eq $name} {
         }
     }
 
-    variant openssl description {Enable OpenSSL support} {
-        depends_lib-append      path:lib/libssl.dylib:openssl
-
-        patchfiles-delete       patch-cmake-install_layout.cmake-yassl.diff
-
-        configure.args-delete   -DWITH_SSL:STRING=bundled
-        configure.args-append   -DWITH_SSL:STRING=${prefix}
```

Comment:
Is this no longer required ?

Reply:
Yes, the port already appends the `-DWITH_SSL:STRING=${prefix}` argument, and no longer has a `-DWITH_SSL:STRING=bundled` argument to delete. The patchfile no longer has any effect because there is no bundled yassl to build.